app/utils/url_utils.rb

```diff
@@ -43,4 +43,13 @@ def extract_locale_from_url(url)
   def strip_port_from_host(host)
     host.split(":").first
   end
+
+  # Naive join method, which can be used to normalize multiple slashes
+  #
+  # Usage: URLUtils.join("foo", "bar/", "baz") => "foo/bar/baz"
+  def join(*parts)
+    parts.select(&:present?).reduce { |a, b|
+      "#{a.sub(/\/+\z/, "")}/#{b}"
```

Comment:
@bladealslayer Yeah, I thought about that as well, but the problem with `File.join` is that the result depends on the OS. It's `foo/bar` for Unix and `foo\bar` for Windows.